src/Console/Commands/MakeDataGrid.php

```diff
@@ -0,0 +1,117 @@
+<?php
+
+namespace WdevRs\LaravelDatagrid\Console\Commands;
+
+use Illuminate\Support\Str;
+use Symfony\Component\Console\Input\InputArgument;
+
+class MakeDataGrid extends \Illuminate\Console\GeneratorCommand
+{
+    /**
+     * The name and signature of the console command.
+     *
+     * @var string
+     */
+    protected $signature = 'make:datagrid {name} {--M|model=YourModel} {--F|fields=yourfield1,yourfield2}';
+
+    /**
+     * The console command description.
+     *
+     * @var string
+     */
+    protected $description = 'Create a new datagrid';
+
+    /**
+     * The type of class being generated.
+     *
+     * @var string
+     */
+    protected $type = 'DataGrid';
+
+    /**
+     * Replace the class name for the given stub.
+     *
+     * @param string $stub
+     * @param string $name
+     * @return string
+     */
+    protected function replaceClass($stub, $name): string
+    {
+        $className = $this->argument('name');
+        $model = $this->option("model");
```

Comment:
Would it be possible to generate a use statement to the top of the created datagrid file for the given model with the default namespace, eg: use App\Models\{model given as argument}?